src/GraphQL/SchemaExtensions.cs

```diff
@@ -216,19 +216,19 @@ public static void Run(this ISchemaNodeVisitor visitor, ISchema schema)
                         visitor.VisitObject(output, schema);
                         foreach (var field in output.Fields.List)
```

Comment:
Since all visit methods are virtual one can override them to fo nothing and NRE may be here. Need to write `  if (output.Fields?.Count > 0)` here and for `IInputObjectGraphType` and even `EnumerationGraphType` cases.

Reply:
Well, I see `private` ctor, OK. But let's do check anyway.

Reply:
Nevermind. `TypeFields` always initializes inner list.